homework-submission/myapp/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import "./App.css";
+import Forms from "./components/forms";
+import TodoItems from "./components/TodoItems";
+import { inject, observer } from "mobx-react";
+import Header from "./components/header";
+
+@inject("RashadStore")
+@observer
```

Comment:
Depending on the Application size you have and the children levels it's Okay to `inject` on the top level of your app and pass it as props.

if you are in a real world Or your prject and because of a multy developers may work on the same file.
ONLY in case you have a one or two levels deep you can pass it other wise you are using MobX wasting your desk memory, and your team time following the code 🙃 .